ghost/admin/app/components/posts-list/context-menu.js

```diff
@@ -137,6 +139,30 @@ export default class PostsContextMenu extends Component {
         this.updateFilteredPosts();
 
         close();
+    }
+
+    @task
+    *featurePostsTask() {
+        const updatedModels = this.selectionList.availableModels;
+        yield this.performBulkEdit('feature');
+
+        // Update the models on the client side
+        for (const post of updatedModels) {
+            post.set('featured', true);
+        }
+
+        return true;
+    }
+
+    @task
+    *unfeaturePostsTask() {
+        const updatedModels = this.selectionList.availableModels;
+        yield this.performBulkEdit('unfeature');
+
+        // Update the models on the client side
+        for (const post of updatedModels) {
+            post.set('featured', false);
+        }
```

Comment:
```suggestion
        // Update the models on the client side
        for (const post of updatedModels) {
            // We need to do it this way to prevent marking the model as dirty
            this.store.push({
                data: {
                    id: post.id,
                    type: 'post',
                    attributes: {
                        featured: false
                    }
                }
            });
        }
```

Reply:
```
// Remove posts that no longer match the filter
this.updateFilteredPosts();
``` 
Is also missing at the end